Traceback (most recent call last):
  File "/Users/avarela/anaconda3/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/avarela/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/avarela/anaconda3/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/avarela/anaconda3/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/avarela/anaconda3/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Users/avarela/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "/Users/avarela/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/avarela/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from sklearn.model_selection import train_test_split 

# Reading al categories but class from 1 to end
X= df_mushroom.drop(["class"],axis=1)
#Â Reading classes (edible vs poisonous)
y= df["class"] 

# Spliting dataset in training (70%) and evaluation (30%) using 0 as a seed
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.30, random_state=0)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-3-c9668e99b240>[0m in [0;36m<module>[0;34m[0m
[1;32m      4[0m [0mX[0m[0;34m=[0m [0mdf_mushroom[0m[0;34m.[0m[0mdrop[0m[0;34m([0m[0;34m[[0m[0;34m"class"[0m[0;34m][0m[0;34m,[0m[0maxis[0m[0;34m=[0m[0;36m1[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0;31m#Â Reading classes (edible vs poisonous)[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 6[0;31m [0my[0m[0;34m=[0m [0mdf[0m[0;34m[[0m[0;34m"class"[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      7[0m [0;34m[0m[0m
[1;32m      8[0m [0;31m# Spliting dataset in training (70%) and evaluation (30%) using 0 as a seed[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 'df' is not defined
NameError: name 'df' is not defined

